src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -731,19 +728,6 @@ func (c controller) waitingForClusterVersion() error {
 	return nil
 }
 
-func (c controller) sendCompleteInstallation(isSuccess bool, errorInfo string) {
-	c.log.Infof("Start complete installation step, with params success:%t, error info %s", isSuccess, errorInfo)
-	for {
-		ctx := utils.GenerateRequestContext()
-		if err := c.ic.CompleteInstallation(ctx, c.ClusterID, isSuccess, errorInfo); err != nil {
```

Comment:
What do you mean it does nothing?
The assisted-service CompleteInstallation API is no op?